### ISSUE TITLE
Keyboard shortcuts stop working after re-opening tab

### ISSUE THREAD

**Issue by dbeckwith** [jonathandelgado/SublimeTodoReview#164]
If I have a TodoReview tab open when I close a project, then open the project back up again, the content of the TodoReview tab is restored, but the keyboard shortcuts no longer work, and it just behaves like a plain text tab.

**Comment by jonathandelgado** [comment]
Yeah, unfortunately this is working as expected. There isn't a way I'm aware of (this may have changed) to serialize that data in a way where it can be restored with the window. If you know of any way to do this, I would be happy to accept a PR!

**Comment by FichteFoll** [comment]
This is curious. I don't know how this is implemented currently, but ST preserves view-specific settings across sessions.
